ctr/const.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+// ctr wide constants
+const (
+	// EXIT_STATUS_OK indicates successful completion
+	EXIT_STATUS_OK = 0
+
+	// EXIT_STATUS_MISSING_ARG indicates failure due to missing argument(s)
+	EXIT_STATUS_MISSING_ARG = 1
```

Comment:
I find that Golang use camel case for constant too, https://golang.org/src/net/http/status.go